css/src/components/steps.scss

```diff
@@ -0,0 +1,23 @@
+$step-font-size: $font-size-7 !default;
+$steps-gap: 0.4375em !default;
+$steps-size: 0.5em !default;
+$steps-size-double: 2 * $steps-size;
```

Comment:
With the `aspect-ratio` addition below, this variable isn't used anymore.

Reply:
thanks, removed!